completions/zsh/_brew

```diff
@@ -442,11 +440,11 @@ _brew_install() {
       '(--devel --HEAD)'{--devel,--HEAD}'[install the development / HEAD version]' \
       '--keep-tmp[don''t delete temporary files created during installation]' \
       '--display-times[display installation times at end of run]' \
-      '*: : __brew_formulae' \
+      '*: : __brew_formulae_or_ruby_files' \
```

Comment:
Let's not handle it for installation or any non-development commands as it doesn't work quite as expected (i.e. it won't be upgraded properly).

Reply:
Will do! I added it because when I'm testing i'm always doing install from a formula, but I'm happy to trim it down if that would cause problems for people.

Reply:
Thanks!

Reply:
I've updated this to cut back to only the commands that I think are used are development related.  Please let me know if you if you want me to make additional changes.  I wasn't certain about some commands.